src/TicketMap/index.tsx

```diff
@@ -391,66 +391,60 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
   }
 
   /**
-   * Interation Callbacks
+   * Interaction Callbacks
    */
 
-  onMouseOver = (event: React.MouseEvent<HTMLElement>) => {
-    const element = (event.target as HTMLElement).closest('[data-section-id]')
-    if (!element) {
-      return
-    }
+  onMouseOver ({ clientX, clientY, target }: React.MouseEvent<HTMLElement>) {
```

Comment:
These handlers aren't bound to the component, so they'll fail when attempting to access properties and methods of `this`

They need to be bound in the constructor or written using the bound method syntax:
```suggestion
  onMouseOver = ({ clientX, clientY, target }: React.MouseEvent<HTMLElement>) => {
```